examples-generated/cloudplatform/project.yaml

```diff
@@ -10,6 +10,7 @@ spec:
   forProvider:
     name: My Project
     orgId: "1234567"
+    projectId: your-project-id
```

Comment:
The `examples/cloudplatform/project.yaml` is missing the required `name` parameter. We may consider adding it in this PR.

Reply:
Good idea, done.

Reply:
Thank you @Pocokwins.